### PR TITLE
test(admin): fixing the failing E2E tests

### PR DESCRIPTION
**Pull request by HejdaJakub**
* New path to advanced settings (and related logic) broke some tests, so it it fixed now.
* Removed skip from tests which were failing due to the bug with privileges because this bug has been already fixed IMHO (I think it was related to the refreshing roles on BE where we have done some changes to avoid PrivilegeException).
* Fixed some 'hidden' items covered by another item (like header).

### CODE REVIEW

**Comment by HejdaJakub** [comment]
- It needs to be tested with this version of containers: https://gitlab.ics.muni.cz/perun/e2e-perun/-/merge_requests/28.
- This needs to be also included during the testing: https://github.com/CESNET/perun-web-apps/pull/1415 (I will rebase it after it will be merged).
- Until this is not released (https://github.com/CESNET/perun/pull/4062) and used as a backend, the last test (for blocked logins) in the perun-observer file will still fail.

File: apps/admin-gui-e2e/src/e2e/facilities/facility-admin.cy.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ describe('Facility management with role Facility admin', () => {
 
   before(() => {
     cy.login('FACILITY_MANAGER', 'facilityManager');
+    window.localStorage.setItem('settings', 'true');
```
Comment (xflord) [inline apps/admin-gui-e2e/src/e2e/facilities/facility-admin.cy.js:16]: The storage items could be set before all the tests to save code, right? I believe support/e2e.ts is the place for this

**Review by xflord — COMMENTED** [review]
will test locally and see what happens

**Review by xflord — APPROVED** [review]
everything passes, good job

**Comment by HejdaJakub** [comment]
Good point, it works fine. The setting of the localStorage items has been moved to support/e2e.ts file.
I noticed that vo-management/perun-admin test has incorrect title in describe block, so I changed it.
Agreed that all tests now passing.
image

**Review by mattjoke — APPROVED** [review]
Everything seems to pass. Good job :+1: 
image

**Review by xflord — APPROVED** [review]

**Pull request merged**